telluric/vectors.py

```diff
@@ -343,6 +349,17 @@ def from_xyz(cls, x, y, z):
                                xmax=bb.right, ymax=bb.top,
                                crs=WEB_MERCATOR_CRS)
 
+    def __or__(self, other):
+        return self.union(other)
+
+    def __and__(self, other):
+        return self.intersection(other)
+
+    def __add__(self, other):
```

Comment:
do we need this method? I don't find it intuitive, people may assume it is a synonym of `or`

Reply:
do you mean that people might confuse `vec1 + vec2` with `vec1 | vec2`?

Reply:
I think so,  but it is my personal opinion